catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -50,11 +51,71 @@ class NodeViewTableViewDemoCell: UITableViewCell {
   }
 }
 
+class NodeViewTableViewPrimaryDemoCell: UITableViewCell {
+
+  let containedButton = MDCButton()
+
+  override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+    setupContaintButton()
+  }
+
+  required init(coder: NSCoder) {
+    super.init(coder: coder)!
+  }
+
+  override func prepareForReuse() {
+    super.prepareForReuse()
+  }
+
+  func setupContaintButton() {
+    containedButton.setTitle("Start Demo", for: .normal)
+    let buttonScheme = MDCButtonScheme()
```

Comment:
Use the app's global theme (AppTheme.globalTheme).

Reply:
Done.